cves/kernel/CVE-2021-29647.yml

```diff
@@ -322,8 +315,8 @@ vouch:
 
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of what your answer was.
-  answer:
-  note:
+  answer: false
+  note: This was a simple fix no vouching occurred.
```

Comment:
This is not true. The commit that fixed the issued was signed off by at least one person that I could find.